plugins/modules/pnp_intent.py

```diff
@@ -352,15 +359,17 @@ def get_site_type(self):
                 params={"name": self.want.get("site_name")},
             )
         except Exception:
+            self.log("Site {0} not found".format(self.want.get("site_name")), "CRITICAL")
             self.module.fail_json(msg="Site not found", response=[])
 
         if response:
-            self.log(str(response))
+            self.log("Site details of {0} received are {1}".format(self.want.get("site_name"), str(response)), "DEBUG")
```

Comment:
self.log("Received site details for '{0}': {1}".format(self.want.get("site_name"), str(response)), "DEBUG")

Reply:
Resolved